llvm/lib/Transforms/IPO/DynUntrustedAllocPost.cpp

```diff
@@ -171,6 +178,14 @@ class DynUntrustedAllocPost : public ModulePass {
     return true;
   }
 
+  // Reference for this function can be found in the LLVM JSON support 
+  // library documentation, `llvm/include/llvm/Support/JSON.h` titled:
+  // "Converting JSON values to C++ types". This describes a function: 
+  // fromJSON(const json::Value&, T&) -> bool
+  // which passes the JSON object as its first argument and a reference
+  // to the data structure you are deserializing as the second argument.
```

Comment:
This is fine, but a little over the top. no need to repeat documentation if it is elswhere. The one exception is describing the caller's responsibility.

nit:`T/F` should be `boolean`

Reply:
Made change to `boolean` as suggested.